src/server/ui.js

```diff
@@ -10,8 +10,8 @@ const path = require('path');
 const React = require('react');
 const morgan = require('morgan');
 const { Subject } = require('rxjs');
-const { renderToTerminal, Text, ProgressBar } = require('react-slate');
-const { hideCursor, overwriteConsole } = require('react-slate-utils');
+const { renderToTerminal, Text, ProgressBar } = require('@react-slate/core');
```

Comment:
`ProgressBar` is a part of `@react-slate/components` package now, so you need to install it as well 😉

Reply:
Good catch! Fixed.